src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/FileContentProcessorTest.java

```diff
@@ -35,11 +35,12 @@
     private static final String FILE_NAME = "file1.zip";
     private static final String CONTAINER_NAME = "container";
     private static final String LEASE_ID = "leaseID";
-    private static final String DCN = "dcn";
     private static final String POBOX = "pobox";
     private static final String BULKSCAN = "bulkscan";
     private static final String CASE_NUMBER = "case_number";
     private static final String CASE_REFERENCE = "case_reference";
+    private static final Long EVENT_ID = 1L;
+    public static final String MSG = "msg";
```

Comment:
Can we keep this change outside this PR? I don't agree with creating static variables without any particular meaning rather than being used multiple times as any value. We can move that discussion into another PR, not to block this one.